test/inductor/test_max_autotune.py

```diff
@@ -439,6 +439,137 @@ def mm(a, b):
         with config.patch({"max_autotune": True}):
             torch.compile(mm, dynamic=dynamic)(a, b)
 
+    @unittest.skipIf(
+        not has_datacenter_blackwell_tma_device(),
+        "Need Blackwell with device-side TMA support in Triton",
+    )
+    @parametrize("dynamic", (False, True))
+    @parametrize("tma_store", (False, True))
```

Comment:
Nit: I'm assuming we don't have support for anything but row-wise A/B yet. I think that's fine and can be done as a followup, but let's add a TODO to include that.

Reply:
Also adding a note to ensure we are checking that in the template making decisions.